src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  background-color: #000;
+  padding: 0 12px;
+}
+
+.first,
+.third,
+.second {
+  /* box-sizing: border-box; */
+  background-color: #ddd;
+  border: 1px solid #e95d49;
+  padding: 10px;
+  margin: 0;
+}
+
+.middleBox {
+  box-sizing: border-box;
+  background-color: #ffa500;
+  padding: 10px;
+  border: 1px solid #e95d49;
```

Comment:
You have same border and padding styles in three places.
Don't repeat yourself, create a common class for these styles.